docs/GettingStartedDocs/buildandsign.md

```diff
@@ -133,3 +135,50 @@ specify a `CONFFILE` that only changed that property:
 # Enclave settings:
 Debug=0
 ```
+
+## Signing an OP-TEE Enclave
+
+The `oesign` tool is currently SGX-only.  For OP-TEE enclaves, signing
+is instead performed by the `sign.py` script that comes with OP-TEE, and the
+script uses a key that is integrated into OP-TEE itself.  The signing key
+and script, among other artifacts, are exported to a "TA Dev Kit" during
+OP-TEE's build process. The Open Enclave SDK takes a CMake parameter at
+build time that specifies where to find the TA Dev Kit. This enables
+the `add_enclave` CMake function to locate sign.py and the TA signing key
+and apply them automatically to every TA built as part of the build process.
+
+There is currently no equivalent of a .conf file for OP-TEE, so all settings
+must be done using the `OE_SET_ENCLAVE_OPTEE` macro:
+
+```c
+#define TA_UUID                                            \
+    { /* 126830b9-eb9f-412a-89a7-bcc8a517c12e */           \
+        0x126830b9, 0xeb9f, 0x412a,                        \
+        {                                                  \
+            0x89, 0xa7, 0xbc, 0xc8, 0xa5, 0x17, 0xc1, 0x2e \
+        }                                                  \
+    }
+
+OE_SET_ENCLAVE_OPTEE(
+    TA_UUID,               /* UUID */
+    2 * 1024 * 1024,       /* Heap size, in bytes */
+    24 * 1024,             /* Stack size, in bytes */
+    TA_FLAG_MULTI_SESSION, /* Flags */
+    "1.0.0",               /* Version */
+    "Sample enclave");     /* Description */
+```
+
+The flags property can contain any of the flags defined in the
+`include/user_ta_header.h` file in the TA Dev Kit:
+
+| Flag                        | Meaning                                     |
+| :-------------------------- | :------------------------------------------ |
+| TA_FLAG_SINGLE_INSTANCE     | all host apps use the same enclave instance |
+| TA_FLAG_MULTI_SESSION       | allow multiple sessions from host apps      |
+| TA_FLAG_INSTANCE_KEEP_ALIVE | keep enclave running after sessions end     |
+| TA_FLAG_SECURE_DATA_PATH    | accesses SDP memory                         |
+| TA_FLAG_REMAP_SUPPORT       | use map/unmap syscalls                      |
```

Comment:
`TA_FLAG_REMAP_SUPPORT` is marked as deprecated in [user_ta_header.h](https://github.com/OP-TEE/optee_os/blob/master/lib/libutee/include/user_ta_header.h), should it still be mentioned here? 

@HernanGatta Similarly, everywhere this macro is used in OE SDK today, the flag `TA_FLAG_EXEC_DDR` is specified , which may be confusing since that is also marked as deprecated already (and may be confusing since it's not documented in this table).

Reply:
I had noticed the DDR issue when generating the PR and omitted that.  Yes I think it should be removed from all of our code (which isn't a doc issue so didn't do it in this PR).   I hadn't noticed that REMAP_SUPPORT was also deprecated, I'll remove that from this PR accordingly.  Thanks for pointing this out.

Reply:
Done

Reply:
I agree we should remove `TA_FLAG_EXEC_DDR` from the existing code; it used to be a mandatory flag.